client/src/components/thumbnailimage.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Link } from 'react-scroll';
+
+const ThumbnailImage = ({ image }) => {
+  const images = JSON.parse(image.photo_urls);
+  // console.log(Array.isArray(image))
+  return (
+    <div className="thumbnailContainer">
+      {images.map((url) => {
+        return (
+          <div>
+            <Link
+              activeClass="active"
+              to={url}
+              spy={true}
+              smooth={true}
+              offset={-70}
+              duration={300}
+            >
+              <img className="thumbnailImages" src={url}></img>
```

Comment:
I'm not familiar with react-scroll, so no crits there, but couldn't your `<img>` tag be combined to be `<img />`?